Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocTextValue.css

```diff
@@ -7,31 +7,31 @@ span.bocTextValue
   position: relative;
   display: inline-block;
   vertical-align: middle;
-  overflow: hidden;
+}
+
+span.bocTextValue textarea
+{
+  /* Otherwise the textarea is slighty offset in Firefox since a margin is added there. */
+  margin: 0;
 }
 
 .bocTextValue.readOnly
 {
   min-height: 0;
-  /* Ensures that the outline is visible in Firefox. */
-  overflow: visible;
 }
 
-.bocTextValue.disabled
+span.bocTextValue .content,
+span.bocTextValue input,
+span.bocTextValue textarea
 {
+  box-sizing: border-box;
+  width: 100%;
+  height: 100%;
 }
```

Comment:
I just re-checked with the styleguide / screenshots and looked at the live styles: the designers would like to have a 0.3-0.5em padding between the textbox border and the text.
![image](https://user-images.githubusercontent.com/6972548/129323787-9da425df-d6ec-42c7-8913-fca251bdd97a.png)
I think we should be golden by adding a padding-left and padding-right of 0.4em and setting the height of the input's parent the to min-height + 0.8em. Possibly, it would be a good idea to use calc for the height, that way, we can take the intended height and add 2*padding to it and thus make it clear where the dimensions come from. For textarea, we have to use padding all around. 
The padding/height is to be globally defined, probably also part of the global input[text,passowrd] element style.

Please make sure the text is properly centered vertically in the textbox.

The same styling applies to readonly elements, probably will need their own considerations about layout.

Reply:
Since padding is now global for input, textarea, and readonly elements, the value has been set to 0.3em instead of 0.4 to avoid scroll bars for BocMultilineTextValue-textareas since the padding is applied there as well (~2px too large if 0.4em). In general, it still doesn't look _exactly_ like the designers' concept, however it is way closer now than before.

Reply:
I'm not married to 0.4 vs 0.3, but I don't get the scrollbar bit. If the textarea is not high enough, we add height.

Could you do a screenshot-compare of the designer-provided template and our control, please? If we use the Roboto-font and make sure the textsize in EM alines to the one in PX, we should be able to determine if 0.3em or 0.4em better matches the distances. It doesn't need to be pixel-perfect but the ratio should be close and I've never seen the two side by side..

Reply:
Since the size of the textarea can simply be adjusted if necessary, I now went with the `0.4em` as initially suggested.

Comparison/screenshots with temporarily applied `Roboto` font:

# Facelift
## Firefox
### Currently
![new_firefox](https://user-images.githubusercontent.com/62145161/129697566-9b8dc6d3-cc32-4f55-8fa7-41f9423508c1.png)

### With Roboto
![new_firefox_roboto](https://user-images.githubusercontent.com/62145161/129697580-1ae1264b-264d-43fb-b9e8-7b34be1d4e1d.png)

### With Roboto and temporarily adjusted FromGrid cell padding to imitate intended design
![new_firefox_roboto_cell_changed](https://user-images.githubusercontent.com/62145161/129697596-e3fd0382-a0dc-422e-8069-ac539681785a.png)


## Chrome
### Currently
![new_chrome](https://user-images.githubusercontent.com/62145161/129697606-9ee5ae95-f3a9-4db4-9ab7-edc7d0a50b54.png)

### With Roboto
![new_chrome_roboto](https://user-images.githubusercontent.com/62145161/129697621-e96b3143-4564-4db9-a485-00a86134983a.png)

### With Roboto and temporarily adjusted FromGrid cell padding to imitate intended design
![new_chrome_roboto_cell_changed](https://user-images.githubusercontent.com/62145161/129697628-16d784f2-faa1-4c85-86fa-135f9e3e5fc5.png)


# Zeplin
## Example 1
![zeplin1](https://user-images.githubusercontent.com/62145161/129697320-7a12ddab-b3cc-429b-b9d1-e9146dbb48ff.png)

## Example 2
![zeplin2](https://user-images.githubusercontent.com/62145161/129697332-85f89e8f-5bd1-4207-b1b6-1f235aaeb08a.png)